src/lib/list/selection-list.ts

```diff
@@ -492,14 +489,14 @@ export class MatSelectionList extends _MatSelectionListMixinBase implements Focu
     return this.options.filter(option => option.selected).map(option => option.value);
   }
 
-  /** Toggles the selected state of the currently focused option. */
-  private _toggleSelectOnFocusedOption(): void {
+  /** Toggles the state of the currently focused option if enabled. */
+  private _toggleFocusedOptionIfEnabled(): void {
```

Comment:
As discussed on Slack. In my opinion, this would result in code duplication that can be avoided by just having such a method. 

 I'm not sure about the method name, though. I agree that `IfEnabled` sounds weird. For now, changed it to just `_toggleFocusedOption`.